src/containers/ImportResources/ImportResources.scss

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.bx--text-input__field-wrapper {
+  width: 100%;
+}
+
+.saDropdown {
+  margin: 0rem !important;
```

Comment:
please remove `!important`, if you need to override existing styles use a more specific selector e.g. `.someParentElement .saDropdown`

units can be omitted for `0`